src/webview_provider/docsEditPanel.ts

```
@@ -649,270 +657,390 @@
+      const project = this.getProject();
+      if (project === undefined) {
+        return undefined;
+      }
+
+      if (!patchPath) {
+        switch (message.dialogType) {
+          case "Existing file":
+            const openDialog = await window.showOpenDialog({
+              filters: { Yaml: ["yml"] },
+              canSelectMany: false,
+            });
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)